deeprank/generate/DataGenerator.py

```diff
@@ -118,7 +118,7 @@ def __init__(self,pdb_select=None,pdb_source=None,pdb_native=None,pssm_source=No
         # filter the cplx if required
         self.pdb_path = self.all_pdb
         if self.pdb_select is not None:
-            self.pdb_path = list(filter(lambda x: self.pdb_select in x, self.all_pdb))
+            self.pdb_path = list(filter(lambda x: any(map(lambda i: i in x, self.pdb_select)), self.all_pdb))
```

Comment:
I guess that's working but as a small comment complex one liner should be avoided for readability. I do the same thing though so I know I'm guilty :)

Reply:
Thanks, let me make it simpler.

Reply:
Done.